Database/Terraform/main.tf

```diff
@@ -0,0 +1,68 @@
+# This file terraforms the database on AWS RDS
+
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.AWS_REGION
+}
+
+# terraform import aws_db_instance.database c11-hermes-db
+resource "aws_db_instance" "database" {
+    allocated_storage                     = 20
+    db_name                               = var.DB_NAME
+    availability_zone                     = "eu-west-2a"
```

Comment:
I would also put this in `tfvars` as it depends on `var.AWS_REGION`.

Reply:
done